types/react-table/react-table-tests.tsx

```diff
@@ -23,7 +23,7 @@ const columns: Column[] = [
   {
     Header: 'Stats',
     columns: [
-      { Header: "Visits", accessor: "visits" }
+      { Header: "Visits", accessor: (row: any) => row.visits }
```

Comment:
This test would actually work even without the change, due to the explicit type annotation of `: any` being compatible with anything.